models/DataObject/Objectbrick/Definition.php

```diff
@@ -739,7 +736,7 @@ protected function doEnrichFieldDefinition($fieldDefinition, $context = [])
      */
     public function isWritable(): bool
     {
-        if (getenv('PIMCORE_CLASS_DEFINITION_WRITABLE')) {
+        if (($_SERVER['PIMCORE_CLASS_DEFINITION_WRITABLE'] ?? 'false') === 'true') {
```

Comment:
Should we just do a `(bool) ($_SERVER['PIMCORE_DEV_MODE'] ?? false)` here, as we also do in other places? 
See e.g. https://github.com/pimcore/pimcore/blob/a35a1544b969e049489cb29054673542b4c70025/lib/Pimcore.php#L62, also Symfony is doing that for boolean switches: 
https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/5.2/public/index.php#L18

Reply:
Can change this. But than i would change the doc to PIMCORE_CLASS_DEFINITION_WRITABLE=1/0 because true/false is both true.

Reply:
Yes, I think that would make sense, existing `true` will continue to work as well.